ShowBuildMenu.bat

```diff
@@ -2,14 +2,15 @@
 pushd %~dp0
 
 set NANT="%~dp0Tools\nant\bin\NAnt.exe" -t:net-4.0
-set BUILD_TOOL_PATH=%~dp0Tools\BuildTool\BuildTool.csproj
-set BUILD_TOOL_SLN_PATH=%~dp0Tools\BuildTool\BuildTool.sln
-set BUILDTOOL=dotnet run -p %BUILD_TOOL_PATH% -c Release --no-build
+set BUILD_TOOL_PATH=%~dp0Tools\BuildTool\bin\BuildTool.dll
+set BUILDTOOL=dotnet %BUILD_TOOL_PATH%
```

Comment:
Be it under Linux or Windows, I was noticing a lag in the build menu for displaying the letter choices prompt. And there was also a startup lag due to relaunching the build.

There was also ome differences in the way the build tool was handled between Linux and Windows, without me seeing why they would need to be differently handled.

So I have changed the logic for building only if the binary is not already there, and for running directly the binary (`dotnet pathOfBinary`) instead of running the project from sources (`dotnet run pathToSrc`).

This eliminates the lag in the build menu both under Linux and Windows.

For simplicity, I have redirected the build output directly in the bin folder instead of letting it go into `Release\netcoreapp2.0`.